src/applications/edu-benefits/1995/containers/StemEligibilityView.jsx

```diff
@@ -0,0 +1,246 @@
+import React from 'react';
+import _ from 'lodash';
+import { connect } from 'react-redux';
+import classNames from 'classnames';
+import { isChapter33 } from '../helpers';
+
+export class StemEligibilityView extends React.Component {
+  onChange = property => {
+    this.props.onChange({
+      ...this.props.formData,
+      ...property,
+    });
+  };
+
+  icon = indication => (indication ? 'fa fa-check' : 'fa fa-times');
+
+  iconColor = indication =>
+    indication ? 'vads-u-color--green' : 'vads-u-color--gray-medium';
+
+  renderChecks = () => {
+    const {
+      isEdithNourseRogersScholarship,
+      benefit,
+      exhaustionOfBenefits,
+      exhaustionOfBenefitsAfterPursuingTeachingCert,
+      isEnrolledStem,
+      isPursuingTeachingCert,
+    } = this.props;
+
+    const isEdithNourseRogersScholarshipCheck =
+      isEdithNourseRogersScholarship && isChapter33({ benefit });
+    const isEdithNourseRogersScholarshipQuestion =
+      isEdithNourseRogersScholarship && benefit === undefined;
+    const exhaustionOfBenefitsCheck =
+      exhaustionOfBenefits || exhaustionOfBenefitsAfterPursuingTeachingCert;
+    const isEnrolledStemCheck = isEnrolledStem || isPursuingTeachingCert;
+
+    const recipientIcon = isEdithNourseRogersScholarshipQuestion
+      ? 'fa fa-question'
+      : this.icon(isEdithNourseRogersScholarshipCheck);
+
+    const isEdithNourseRogersScholarshipClasses = classNames(
+      recipientIcon,
+      this.iconColor(isEdithNourseRogersScholarshipCheck),
+    );
+    const exhaustionOfBenefitsClasses = classNames(
+      this.icon(exhaustionOfBenefitsCheck),
+      this.iconColor(exhaustionOfBenefitsCheck),
+    );
+    const isEnrolledStemClasses = classNames(
+      this.icon(isEnrolledStemCheck),
+      this.iconColor(isEnrolledStemCheck),
+    );
+
+    return (
+      <div>
+        <p>
+          <span className="vads-u-font-family--serif heading-level-4">
+            Based on you responses, it appears you're not eligible.
+          </span>
+          <br />
+          <br />
+          <b>Your responses:</b>
+        </p>
+        <ul className="fa-ul stem-eligibility-ul">
+          <li>
+            <span className="fa-li">
+              <i
+                className={isEdithNourseRogersScholarshipClasses}
+                aria-hidden="true"
+              />
+            </span>
+            Post-9/11 GI Bill beneficiary or Fry Scholarship recipient
+          </li>
+          <li>
+            <span className="fa-li">
+              <i className={exhaustionOfBenefitsClasses} aria-hidden="true" />
+            </span>
+            Have used all your education benefits or are within 6 months of
+            doing so
+          </li>
+          <li>
+            <span className="fa-li">
+              <i className={isEnrolledStemClasses} aria-hidden="true" />
```

Comment:
This icon appears to be semantic (either a check or an x).  From [FontAwesome's accessibility docs](https://fontawesome.com/v4.7.0/accessibility/):

> If you're using an icon to convey meaning (rather than only as a decorative element), ensure that this meaning is also conveyed to assistive technologies.

Please consider how you want to convey the icon's information to a screenreader.

Reply:
Added a `title` property to all 3 icons that changes based on what icon is displayed